modules/openideal_challenge/openideal_challenge.module

```diff
@@ -23,6 +25,19 @@ function openideal_challenge_cron() {
   $challenge_service->closeChallenges();
 }
 
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ */
+function openideal_challenge_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (isset($form['created']['widget'][0]['value'])) {
+    $value = &$form['created']['widget'][0]['value'];
+    $date_format = DateFormat::load('openideal_date')->getPattern();
+    $value['#attributes']['data-date-format'] = 'DD/MM/YYYY';
```

Comment:
Is there any way to use the format from field settings?

Reply:
I didn't found it. There is #date_time_format, but it works only for Safari.